### PR TITLE
Calvin

### PR DESCRIPTION
**Pull request by ecols1999**

### CODE REVIEW

File: Sprint 2/ClearScreen.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import java.io.IOException;
```
Comment (ecols1999) [inline Sprint 2/ClearScreen.java:1]: For all of your code, I think that you should provide the author of the code and a short description of what the code does at the top of the page.

File: Sprint 2/DBConnection.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+		
+	    if (connect == null || connect.isClosed()) {
+	        try {
+	            Class.forName("com.mysql.jdbc.Driver");
+	            
+	        } catch (ClassNotFoundException e) {
+	            throw new SQLException(e);
+	        }
+	        
+	        connect = (Connection) DriverManager
```
Comment (ecols1999) [inline Sprint 2/DBConnection.java:28]: I think that this line of code could be made a bit more readable if you have the (Connection) DriverManager and the .getConnection method on its own separate line.

File: Sprint 2/UserDAO.java [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+import java.util.ArrayList;
+import java.util.List;
+public class UserDAO {
+	private Connection connect = null;
+	private Statement statement = null;
+	private PreparedStatement preparedStatement = null;
+	private ResultSet resultSet = null;
+	private DBConnection dbConnection = null;
+	
+	private void createTable() throws SQLException {
```
Comment (ecols1999) [inline Sprint 2/UserDAO.java]: This method would be a lot easier to read if you indent the lines of code properly.

**Review by ecols1999 — COMMENTED** [review]
Here's my code review.

**Review by loganjacksonwsu — APPROVED** [review]
Code looks good to me. However, would changing your Clear Screen method to static change anything for you? When using your ClearScreen method, I changed it to static so I could call it in my classes.

**Pull request merged**